lasagne/layers/base.py

```diff
@@ -1,17 +1,21 @@
 from collections import OrderedDict
 
+import theano
+
 from .. import utils
 
 
 __all__ = [
     "Layer",
     "MergeLayer",
+    "UnLayer",
 ]
 
 
 # Layer base class
 
 class Layer(object):
+
```

Comment:
Please don't insert empty lines where they haven't been any. Either configure your editor to not perform style changes, or remove them before committing.